server/routes/testutils/testUtils.ts

```diff
@@ -602,3 +603,13 @@ export function parseHtmlDocument(documentText: string): Document {
 export function pageHeading(document: Document): string {
   return document.querySelector('[data-qa="page-heading"]')?.textContent
 }
+
+export function expectSomethingWentWrongPage(res: supertest.Response) {
```

Comment:
This isn't functionally the same as the snapshot tests, are we sure we want to remove them?